### PR TITLE
380 from address to account name in the send coins screen

### PR DESCRIPTION
**Pull request by Thykof**
image

### CODE REVIEW

**Comment by Thykof** [comment]
@longgamma-massa low priority, I let you review the pr description screenshot to see if it fit the need

**Comment by longgamma-massa** [comment]
Okay for me as long as we keep the address as well. Did you test if UI breaks with a long wallet account name?

**Comment by Thykof** [comment]
image
I'll try to improve this, @LucasF243 do you have any advice?

File: wails-frontend/src/pages/PasswordPromptHandler/Transaction/Transaction.tsx [hunk 1]
```diff
@@ -26,15 +33,24 @@ export function Transaction(props: PromptRequestData) {
...
           <FiArrowRight size={24} className="text-primary" />
         </div>
         <div className="flex flex-col">
-          <p className="mas-menu-active">{Intl.t('password-prompt.sign.to')}</p>
+          <div className="flex gap-2">
+            <p className="mas-menu-active">
+              {Intl.t('password-prompt.sign.to')}
+            </p>
+            {RecipientAddress ? (
+              <p className="mas-menu-default">
```
Comment (mazmassa) [inline wails-frontend/src/pages/PasswordPromptHandler/Transaction/Transaction.tsx:49]: I believe the `truncate` class achieve the same result with no dependencies. 🤔 

```suggestion
              <p className="mas-menu-default truncate">
```

Comment (Thykof) [reply wails-frontend/src/pages/PasswordPromptHandler/Transaction/Transaction.tsx:49]: no, it doesn't work

**Pull request merged**